moment/moment-node.d.ts

```diff
@@ -317,9 +342,17 @@ declare module moment {
         set(unit: string, value: number): Moment;
         set(objectLiteral: MomentInput): Moment;
         
-        /*This returns an object containing year, month, day-of-month, hour, minute, seconds, milliseconds.*/
-        //Works with version 2.10.5+
+        /**
+         * This returns an object containing year, month, day-of-month, hour, minute, seconds, milliseconds.
+         * @since 2.10.5+
+         */
         toObject(): MomentDateObject;
+
+        /**
+         * @since 2.10.7+
+         */
+
```

Comment:
I think you may need to remove this newline, otherwise the JSDoc annotation might not work for intellisense